GettingStarted.md

```diff
@@ -51,5 +51,7 @@ You will probably want to add `ManualCameraControl.cs` (found under
 
 Deploy to the emulator or device.
```

Comment:
I've now added a new section with regards to deploying the HoloLens app using Visual Studio. Is this fine?